src/main/java/mate/jdbc/Main.java

```diff
@@ -1,8 +1,62 @@
 package mate.jdbc;
 
+import java.util.NoSuchElementException;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final long TEST_CAR_ID = 7L;
+    private static final long TEST_DRIVER_ID = 1L;
+    private static final long TEST_MANUFACTURER_ID = 1L;
+    private static final String TEST_MODEL = "test";
```

Comment:
that's still hardcoded, but okay
i meant something like `carService.delete(testCar.getId())`